config/install/field.storage.node.field_homeworld.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_homeworld
+field_name: field_homeworld
+entity_type: node
+type: entity_reference
+settings:
+  target_type: node
+module: core
+locked: false
+cardinality: -1
```

Comment:
Just realized that the "Homeworld" field on all content types should accept only one value. This will need to be updated in the content model as well. Sorry for missing this until now. 😅
![field_homeworld_cardinality](https://github.com/notS0Solo/swapimigration/assets/1434979/54435f9c-e51e-4048-8aea-68897c927f52)

Reply:
Fixed